Lifestyle/nightscout-graph.50s.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#  <xbar.title>Nightscout Graph and Trend/xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Paul Barrass</xbar.author>
+#  <xbar.author.github>pdaddyo</xbar.author.github>
+#  <xbar.desc>Shows graph and latest data from your glucose sensor via Nightscout.  Requires `brew install fx && brew install gnuplot`</xbar.desc>
+#  <xbar.image>https://github.com/pdaddyo/bitbar-nightscout/assets/7074964/f512b983-cca0-4adc-8583-355d2083ba8b</xbar.image>
```

Comment:
The image URL seems to be specific to the person viewing it. e.g., The above link leads to `https://github-production-user-asset-6210df.s3.amazonaws.com/7074964/332194012-f512b983-cca0-4adc-8583-355d2083ba8b.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=<string>&X-Amz-Date=20240521T110821Z&X-Amz-Expires=300&X-Amz-Signature=<string>&X-Amz-SignedHeaders=host&actor_id=<string>&key_id=0&repo_id=<string>`.

Please upload the image to a publicly available URL.

Reply:
Thanks for reviewing I will update this image URL, there's a bug in the graph when there's missing data that I need to fix too.